x-pack/plugins/security_solution/server/lib/timeline/saved_object/timelines/index.ts

```diff
@@ -132,6 +90,24 @@ export const getTimelineOrNull = async (
   return timeline;
 };
 
+export const resolveTimeline = async (
```

Comment:
We could probably remove this function since it's just a pass through to `resolveSavedTimeline` right?